repository/z.json

```
@@ -200,6 +200,18 @@
 				}
 			]
 		},
+		{
+			"name": "Zmpl",
+			"details": "https://github.com/uzyn/zmpl-grammar",
+			"author": "U-Zyn Chua",
+			"labels": ["zmpl", "zig", "jetzig", "language syntax", "grammar", "html", "markdown"],
```

```suggestion
			"labels": ["zmpl", "zig", "jetzig", "language syntax", "html", "markdown"],
```

They "grammar" keyword is not used in package control, "language syntax" already covers that.

Please also have a look at the bot output: branch based releases are not supported (the docs cover this).

---

Thanks @braver